extensions/wikia/WikiaMobile/templates/WikiaMobileBodyService_index.php

```diff
@@ -34,3 +34,22 @@
 	<div id=wkMdlCnt></div>
 	<div id=wkMdlFtr></div>
 </div>
+<script>
+	// This is quick fix for P2
+	// Ticket https://wikia-inc.atlassian.net/browse/XW-825
+	document.addEventListener("DOMContentLoaded", function() {
+		// Replace widget scripts with iframes
+		$('script[type=x-wikia-widget]').each(function() {
+			$(this).replaceWith(this.textContent);
+		});
+
+		// Show Polldaddy widget
+		$('a[data-wikia-widget=polldaddy]').each(function() {
+			var id = $(this).data('id');
+
+			$(this).replaceWith('<a name="pd_a_' + id + '" style="display:inline;padding:0;margin:0;"></a>' +
```

Comment:
`id` is validated in https://github.com/Wikia/app/blob/dev/extensions/wikia/PolldaddyTag/PolldaddyTagValidator.class.php and must be numeric characters only. No need for escaping.